src/openproduct/producten/urls.py

```diff
@@ -14,7 +14,21 @@
 description = f"""
 Een API voor Producten.
 
+De Producten Api heeft 1 endpoint `producten`. Hiermee kunnen producten zelf, de bijbehorende eigenaren en documenten worden aangemaakt.
+
+### Product
 Een product is een instantie van een producttype (zie producttypen API), in een product worden onder andere de gegevens van de eigenaar, de benodigde data voor het product en bijvoorbeeld de status vastgelegd.
+Specifieke product data kan worden opgeslagen in de json velden `dataobject` en `verbruiksobject`. Deze velden worden gevalideerd door de `verbruiksobject_schema` & `dataobject_schema` velden van het producttype. ([zie jsonschema](https://json-schema.org))
+De status van een product kan alleen worden veranderd naar de een van de `toegestane statussen` gedefineerd op het producttype.
+
+### Eigenaar
+Aan een product kunnen één of meerdere eigenaren worden gelinkt. Een eigenaar kan een klant (Open Klant), natuurlijk of niet natuurlijke persoon zijn.
```

Comment:
The `klant` resource doesn't exist in Open Klant anymore, so I guess it's `Partij`? Not entirely sure what you're able to link to now from Open Product
```suggestion
Aan een product kunnen één of meerdere eigenaren worden gelinkt. Een eigenaar kan een Partij (gedefinieerd in Open Klant), natuurlijk of niet natuurlijke persoon zijn.
```